azure/functions/http.py

```diff
@@ -96,10 +98,17 @@ def encode(cls, obj: typing.Any, *,
                 datum_body = meta.Datum(type='bytes', value=b'')
 
             cookies = None
-            if "Set-Cookie" in headers:
-                cookies = [SimpleCookie(cookie) for cookie in
-                           headers.get_all('Set-Cookie')]
-                headers.pop("Set-Cookie")
+
+            if sys.version_info.major == 3 and sys.version_info.minor <= 7:
```

Comment:
Just checking - there is a helper function - which_python types, somewhere which we've been using in many places. Could you please check?

Reply:
Also - A comment on the why we are skipping would help.

Reply:
It is in the worker, checking for specific version, i prefer checking for 3.7 or below cuz 3.6 is phasing out in 09/22
<img width="434" alt="image" src="https://user-images.githubusercontent.com/12449837/167205379-782f7789-3333-4b99-abf8-fedab9220ec6.png">